l10n_it_fatturapa_out_oss/tests/test_fatturapa_xml_validation.py

```diff
@@ -4,62 +4,60 @@
 import base64
 
 from odoo import fields
+from odoo.tests import Form, tagged
 
-from odoo.addons.l10n_it_fatturapa_out.tests.fatturapa_common import FatturaPACommon
+from .fatturapa_common import FatturaPACommon
 
 
+@tagged("post_install", "-at_install")
 class TestInvoiceOss(FatturaPACommon):
     def setUp(self):
         super().setUp()
 
-        self.today_date = fields.Date.today()
-        self.env.user.company_id.fatturapa_art73 = False
-        au_country = self.env.ref("base.at")
-        self.tax1 = self.env["account.tax"].create(
-            {
-                "name": "OSS AU",
-                "amount": 20.0,
-                "oss_country_id": au_country.id,
-                "kind_id": self.env.ref("l10n_it_account_tax_kind.n3_2").id,
-                "law_reference": "NON IMPONIBILE ART. 41 COMMA 1, LETT. B",
-            }
+        # XXX - a company named "YourCompany" alread exists
+        # we move it out of the way but we should do better here
```

Comment:
@Borruso come mai queste modifiche?
Grazie

Reply:
preso ispirazione dai test in https://github.com/OCA/l10n-italy/tree/16.0/l10n_it_fatturapa_out_di/tests